dist/redhat/scylla.spec

```diff
@@ -11,9 +11,6 @@ Requires:       %{product}-server = %{version}-%{release}
 Requires:       %{product}-conf = %{version}-%{release}
 Requires:       %{product}-python3 = %{version}-%{release}
 Requires:       %{product}-kernel-conf = %{version}-%{release}
-Requires:       %{product}-jmx = %{version}-%{release}
-Requires:       %{product}-tools = %{version}-%{release}
-Requires:       %{product}-tools-core = %{version}-%{release}
```

Comment:
This isn't enough, since we have Wants=scylla-jmx.service in scylla-server.service.

No, Wants= is optional, right? if it's not installed then it won't fail.

Reply:
Wants= is optional, we able to run scylla-server.service w/o installing scylla-jmx.service.